runtime/caches/fileSystem.ts

```diff
@@ -7,10 +7,15 @@ import {
   withCacheNamespace,
 } from "./common.ts";
 import { existsSync } from "std/fs/mod.ts";
+import { LRUCache } from "https://esm.sh/lru-cache@10.2.0";
 
 const FILE_SYSTEM_CACHE_DIRECTORY =
   Deno.env.get("FILE_SYSTEM_CACHE_DIRECTORY") ?? undefined;
 
+const MAX_CACHE_SIZE = parseInt(Deno.env.get("MAX_CACHE_SIZE") ?? "1073741824"); // 1 GB max size of cache
+const TTL_AUTOPURGE = Deno.env.get("TTL_AUTOPURGE") !== "false"; // automatically delete expired items
+const TTL_RESOLUTION = parseInt(Deno.env.get("TTL_RESOLUTION") ?? "30000"); // check for expired items every 30 seconds
```

Comment:
Can you add these variables to the documentation?

I think the repo's `README` would be fine for now

Reply:
Done